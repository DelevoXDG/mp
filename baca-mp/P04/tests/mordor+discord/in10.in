2
32
New A W1
InsertFirst A W0
InsertLast A W2
InsertLast A W3
New B W7
InsertFirst B W6
InsertFirst B W5
InsertFirst B W4
Trains
Display A
Display B
Union A B
Trains
Display A
DelLast A B
DelFirst A C
DelLast A D
DelFirst A E
DelLast A F
DelFirst A G
DelLast A H
DelFirst A I
Trains
Display A
Display B
Display C
Display D
Display E
Display F
Display G
Display H
Display I
34
New A W1
InsertFirst A W0
InsertLast A W2
InsertLast A W3
New B W7
InsertFirst B W6
InsertFirst B W5
InsertFirst B W4
Trains
Display A
Display B
Union A B
Trains
Display A
Reverse A
Display A
DelLast A B
DelFirst A C
DelLast A D
DelFirst A E
DelLast A F
DelFirst A G
DelLast A H
DelFirst A I
Trains
Display A
Display B
Display C
Display D
Display E
Display F
Display G
Display H
Display I