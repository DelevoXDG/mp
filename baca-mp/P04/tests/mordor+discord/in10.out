Trains: B A
A: W0 W1 W2 W3
B: W4 W5 W6 W7
Trains: A
A: W0 W1 W2 W3 W4 W5 W6 W7
Trains: I H G F E D C B
Train A does not exist
B: W7
C: W0
D: W6
E: W1
F: W5
G: W2
H: W4
I: W3
Trains: B A
A: W0 W1 W2 W3
B: W4 W5 W6 W7
Trains: A
A: W0 W1 W2 W3 W4 W5 W6 W7
A: W7 W6 W5 W4 W3 W2 W1 W0
Trains: I H G F E D C B
Train A does not exist
B: W0
C: W7
D: W1
E: W6
F: W2
G: W5
H: W3
I: W4